.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: swift
+osx_image: xcode9.4
```

Comment:
any reason for using older XCode?

Reply:
it's the default on Travis, so I wanted to optimize build times